desserts-recipes/desserts-recipes.js

```diff
@@ -0,0 +1,42 @@
+// access elements
+let container = document.querySelector('.container')
+let input= document.querySelector('.input-value')
+let buttonSearch= document.querySelector('.butt-search')
+
+// fetch food api with end point dessert category 
+fetch(`https://www.themealdb.com/api/json/v1/1/filter.php?c=dessert`).then(response => response.json())
+.then((data)=>{
+    container.innerHTML=" ";
+    let meals=data.meals;
+   
+    meals.forEach((dessert)=>{
+        // create cards
+        let card = document.createElement('div')
+        card.setAttribute('class','card')
+        container.appendChild(card)
+
+        let link = document.createElement('a')
+        card.appendChild(link)
+        //add event listener to open recipe page when click on card
+        link.addEventListener("click", () => {
+            window.location.href = ("#");
+            //save id dessert in local storage
+            saveTolocalStorageCategory(dessert.idMeal);
+          });
+
+        let img= document.createElement('img')
+        img.src= dessert.strMealThumb
+        link.appendChild(img)
+
+        let nameMeal= document.createElement('h3')
```

Comment:
`h3` is too big for the name, on some cards and for meals that consist of more than one word the text is out of the border

<img width="310" alt="Screen Shot 2022-03-22 at 12 52 55 AM" src="https://user-images.githubusercontent.com/47992412/159376098-ec11b35b-5dfb-4869-ad3b-5c006c3c75c1.png">

you can change the font size 

```css
font-size: 0.8rem
```

<img width="310" alt="Screen Shot 2022-03-22 at 12 56 28 AM" src="https://user-images.githubusercontent.com/47992412/159376605-dedc72e5-6b99-4b5f-8172-822033087a76.png">
<img width="924" alt="Screen Shot 2022-03-22 at 12 56 43 AM" src="https://user-images.githubusercontent.com/47992412/159376631-2c8f32c6-4e03-4d45-b0c6-736c1104d479.png">